src/DurableTask.Netherite/Events/PartitionEvents/External/FromClients/CreationRequestReceived.cs

```diff
@@ -75,6 +75,7 @@ public override PartitionEvent Clone()
                     ExtensionData = ee.ExtensionData,
                     OrchestrationInstance = ee.OrchestrationInstance,
                     Timestamp = this.CreationTimestamp,
+                    ParentTraceContext = ee.ParentTraceContext,
                 },
                 SequenceNumber = tm.SequenceNumber,
                 OrchestrationInstance = tm.OrchestrationInstance,
```

Comment:
non blocking question: it appears this copy of the execution started event is very manual, so I worry that we'll break it again in the future if we add a new field. Is there a more future-proof way of doing this copy? For example, via [Object.MemberwiseClone()](https://learn.microsoft.com/en-us/dotnet/api/system.object.memberwiseclone?view=net-7.0)?

If so, perhaps we can add a TODO comment on this line to object copy to consider changing the cloning approach

Reply:
@sebastianburckhardt, what are your thoughts on this?

Reply:
Using `Object.MemberwiseClone` seems like a good idea to me.